README.md

```diff
@@ -265,6 +265,11 @@ Build is based on gradle. See build.gradle included in the repository.
 Changelog
 ---------
 
+### v2.0.0
+
+Updated for jakarta package names for J2EE classes.
+Requires Java 17 now due to dependency on spring-test 6.0.
```

Comment:
Java 17 is only required to test?

If the library can run with Java 11, which is the more commonly found LTS version on servers, I think we should try to require only Java 11 for the library itself even if we need Java 17 for the tests. I think [it can be done with gradle](https://docs.gradle.org/current/userguide/toolchains.html#specify_custom_toolchains_for_individual_tasks). I'm not asking for this to be solved before merging, I'm just asking to know right now.

Reply:
Never mind, I just checked that even the latest Debian 11 ships with Java 17 so it's a reasonable assumption right now. And of course the more time it passes the less of a concern it is. So no need to do extra work here.